utils/run_test.sh

```diff
@@ -63,7 +63,14 @@ function run_test() {
             quay.io/open-cluster-management/grc-policy-framework-tests:$TEST_SNAPSHOT
             ;;
         "CONSOLE_UI")
-            #(TODO)
+           ## The console-ui-tests can be run 2.0.x and 2.1.x
+            sudo $DOCKER run \
+            --volume $result_path:/results \
+            --volume ${config_path}/${_test_case}/options.yaml:/resources/options.yaml \
+            --volume ${config_path}/imported_kubeconfig:/opt/.kube/import-kubeconfig \
+            --env TEST_GROUP=e2e \
+            --env BROWSER='chrome' \
```

Comment:
please try to add --name when start the container since we may want to run multi job on single node in next phase